benchmark/util/priority-queue.js

```diff
@@ -3,10 +3,10 @@
 const common = require('../common');
 
 const bench = common.createBenchmark(main, {
-  n: [1e5]
+  n: [1e6]
```

Comment:
Do we have to increase the value? Our benchmarks have the tendency of taking very long overall.

Reply:
The value is needed to get *** confidence stably. And the benchmark only takes one minute even after the change.